src/gerbil/core/contract.ss

```diff
@@ -635,6 +635,76 @@ package: gerbil/core
        #'(%%app arg ...))))
 
   (defsyntax (%%ref-dotted stx)
+    (def (get-method-sig info method)
```

Comment:
lets lift this as a utility, it is generally useful

Reply:
Where should it go? `(module InterfaceInfo ... )` as `interface-method-input-signature`? 

Along those lines, should it accept a symbol or a pair as the interface method? Makes sense to me.

Reply:
give it the interface info object (or the identifier if you dont have it), no need for a pair when you can use multiple arguments...

Reply:
Added `interface-info-method-input-signature`. Looks good?

Reply:
drop the input 🐈‍⬛

Reply:
But it could have another sig, the output-signature!! 

I get it, name is long, output sig is also return-type ... doing so now.